SimPEG/EM/Static/SIP/Regularization.py

```diff
@@ -1,3 +1,8 @@
+from __future__ import absolute_import
```

Comment:
should this be brought into SimPEG/Regularization.py? I am fine with leaving it in this pr, but would request that you make an issue to bring it in or upgrade it when the objective function stuff from #511 gets in